Formula/r/riemann-client.rb

```diff
@@ -1,8 +1,8 @@
 class RiemannClient < Formula
   desc "C client library for the Riemann monitoring system"
   homepage "https://git.madhouse-project.org/algernon/riemann-c-client"
-  url "https://git.madhouse-project.org/algernon/riemann-c-client/archive/riemann-c-client-2.2.0.tar.gz"
-  sha256 "f690aea02a3652c9dc62372c4305d0b9e869cd8b7d9c22b964991a4bdf95d2be"
+  url "https://git.madhouse-project.org/algernon/riemann-c-client/archive/riemann-c-client-2.2.1.tar.gz"
+  sha256 "65daf32ad043ccd553a8314d1506914c2b71867d24b0e18380cb174e04861303"
   license "LGPL-3.0-or-later"
```

Comment:
```suggestion
  license "EUPL-1.2"
```

was changed in v2.2.0 - https://git.madhouse-project.org/algernon/riemann-c-client/commit/9bada2fabff9124245426baf7beb18e1e9480b17

Reply:
good catch!